build/charts/antrea/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: antrea
+type: application
+displayName: Antrea
+home: https://antrea.io/
+version: 1.17.0-dev
+appVersion: 1.17.0-dev
+kubeVersion: ">= 1.16.0-0"
+icon: https://raw.githubusercontent.com/antrea-io/antrea/main/docs/assets/logo/antrea_logo.svg
+description: Kubernetes networking based on Open vSwitch
+keywords:
```

Comment:
Should we add "OVS"?

Reply:
I'll look into adding keywords in a follow-up PR. I still have to figure versioning & release upload for the Helm chart.